test/blackbox_tests/dns_test.go

```diff
@@ -1027,6 +1028,183 @@ var _ = Describe("Outbound IPv4 DNS/UDP conntrack zone splitting with specific I
 	)
 })
 
+var _ = Describe("Outbound IPv4 DNS/UDP traffic to port 53 from specific input interface", func() {
```

Comment:
We should start thinking about drying up these tests, they're both like ~100 LOC and only like 8 lines are different. Please create an issue for this and add it to the tech backlog

Reply:
https://github.com/kumahq/kuma-net/issues/104